### PR TITLE
onramp: moonpay MVP

### PR DESCRIPTION
**Pull request by x1ddos**
No address or any other kind of secure verification.
We're at the mercy of Moonpay and relying on a non-compromized app
which is kind of against our threat model.

To test this, use:

- working fake CC: 4000 0231 0466 2535, 12/2020, 123
- declined CC: 4008 3708 9666 2369, 12/2020, 123
- TBTC and TETH work; send them back to `tb1q45h8zexwztmz3nyd8gmkxhpavdsva4znwwhzvs` and `0xc216eD2D6c295579718dbd4a797845CdA70B3C36` respectively
- KYC will always succeed; just click on "submit anyway"
- you'll have to provide a valid email addess because they send an check code; I sometimes use https://fakermail.com/ for this

### CODE REVIEW

**Comment by x1ddos** [comment]
Sorry, I didn't provide any way to test this. Updated the PR descr.

**Review by thisconnect — APPROVED** [review]
Nice, LGTM

**Comment by x1ddos** [comment]
Rebased on the most recent master and made tslint happy.

**Review by benma — APPROVED** [review]

**Comment by benma** [comment]
> * working fake CC: 4000 0231 0466 2535, 12/2020, 123
> 
> * declined CC: 4008 3708 9666 2369, 12/2020, 123
> 
> * TBTC and TETH work; send them back to `tb1q45h8zexwztmz3nyd8gmkxhpavdsva4znwwhzvs` and `0xc216eD2D6c295579718dbd4a797845CdA70B3C36` respectively
> 
> * KYC will always succeed; just click on "submit anyway"
> 
> * you'll have to provide a valid email addess because they send an check code; I sometimes use https://fakermail.com/ for this

put this as a comment into the code? will come in handy in the future too.

**Comment by x1ddos** [comment]
Sorry, had to do a couple tiny changes:

1. Allow third party cookies in android. Otherwise, MoonPay refuses to load in the iframe.
2. Added `allow` HTML attribute in frontends/web/src/utils/custom.d.ts for `allow=payment` to *maybe* make android/apple/samsung pay methods work.

**Comment by x1ddos** [comment]
>     1. Otherwise, MoonPay refuses to load in the iframe.

Well, it does load but refuses to do anything useful.

**Comment by x1ddos** [comment]
> put this as a comment into the code? will come in handy in the future too.

done.

I think this should be good to go now. Will merge once CI is done.

File: backend/coins/btc/handlers/handlers.go [hunk 1]
```diff
@@ -486,24 +484,28 @@ func (handlers *Handlers) getHasSecureOutput(r *http.Request) (interface{}, erro
...
+	params := exchanges.BuyMoonpayParams{
+		Fiat: "CHF", // TODO: Get this from app config
+		Lang: "en",  // TODO: Get this from the backend
 	}
-
-	return nil, safello.StoreCallbackJSONMessage(
-		path.Join(handlers.account.FilesFolder(), "safello-buy.json"),
-		message,
-	)
+	//handlers.backend.?I
```
Comment (x1ddos) [inline backend/coins/btc/handlers/handlers.go:496]: damnit. another typo.

**Comment by x1ddos** [comment]
Yay, github's macos CI is finally on go1.14.13 which removed CFLAGS regression introduced  in 1.14.12:
 
image

**Pull request merged**